src/components/Accordion/_Accordion.scss

```diff
@@ -3,15 +3,35 @@
     color: inherit;
     width: 100%;
 
-    &__button {
+    input {
+        position: absolute;
+        opacity: 0;
+        z-index: -1;
+
+        &:checked {
+            + .accordion__label .icon--minus {
+                display: inline;
+            }
+            + .accordion__label .icon--plus {
+                display: none;
+            }
+            ~ .accordion__content {
+                display: inline-block;
+            }
+        }
+    }
+
+    &__label {
         @include button--outline;
 
-        align-items: center;
```

Comment:
Small nit, but by removing this the icon is no longer centered vertically within the accordion label.

Reply:
Nice catch. Updated.